dirty_equals/_boolean.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, Optional, TypeVar
+
+from ._base import DirtyEquals
+
+B = TypeVar('B')
+
+
+class IsFalseLike(DirtyEquals[B]):
+    """
+    Check if the value is False like, and matches the given conditions.
+    """
+
+    def __init__(
+        self,
+        *,
+        allow_strings: Optional[bool] = None,
+    ):
+        """
+        Example of basic usage:
```

Comment:
you need to add `Args` and document what values are allowed when `allow_strings=True`.